src/orquestra/sdk/_base/_logs/_interfaces.py

```diff
@@ -12,8 +12,8 @@
 
 @dataclass(frozen=True)
 class LogOutput:
-    out: t.Sequence[str]
-    err: t.Sequence[str]
```

Comment:
A comment on that change: Its good to have general interface, but we later in the code used + operator which is specifically defined for the lists.